.github/workflows/ci.yaml

```diff
@@ -29,25 +29,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04]
+        os: [ubuntu-22.04, ubuntu-24.04]
         build-type: [Debug, RelWithDebInfo]
-        standalone: [standalone, catkin]
-        solvers-from-source: [SolversFromSource, SolversFromAPT]
+        standalone: [standalone, colcon]
+        solvers-from-source: [SolversFromSource]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Set environment variables
         run: |
-          echo "LD_LIBRARY_PATH=/opt/blasfeo/lib:/opt/hpipm/lib:$LD_LIBRARY_PATH" >> $GITHUB_ENV # for HPIPM
-      - name: Set ROS version
-        if: matrix.standalone == 'catkin'
-        run: |
-          if [ "${{ matrix.os }}" == "ubuntu-20.04" ]
+          set -e
+          set -x
+          echo "LD_LIBRARY_PATH=/opt/blasfeo/lib:/opt/hpipm/lib:/usr/local/lib:$LD_LIBRARY_PATH" >> $GITHUB_ENV # for HPIPM
+          if [ "${{ matrix.os }}" == "ubuntu-22.04" ] && \
+             [ "${{ matrix.build-type }}" == "RelWithDebInfo" ] && \
+             [ "${{ matrix.standalone }}" == "colcon" ] &&  \
+             [ "${{ matrix.solvers-from-source }}" == "SolversFromAPT" ] && \
```

Comment:
Since `SolversFromAPT` has been deleted, this condition is not always satisfied.